src/window-dimensions.ts

```diff
@@ -1,19 +1,32 @@
-import { useEffect, useState } from 'react';
+import { useEffect, useMemo, useState } from 'react';
 import { Dimensions } from 'react-native';
 
 import type { ScaledSize } from 'react-native';
+import { ComponentSizeChanges } from 'make-styles';
```

Comment:
this should be a type import. But also it seems like this file owns this type definition not the other way round, so I'd suggest declaring and exporting it here, rather than the other way round

Reply:
That also avoids the circular dependency which is always a sign of something odd going on